### PR TITLE
Contact page

### LINKED ISSUE #59: UI Design for Contact Us page
**Here is the final design. Please refer to screenshot below.**

contact_us_final

**You can refer to the following image for further details on creating the page :**

contact_us_final_documentation

You can refer to the following image for further details on creating the page :
https://drive.google.com/drive/folders/12WqC1rHA2rakJQLr-FTjZnn4lj0uPXTv

You can get the relevant text from the present website itself: https://evalai.cloudcv.org/contact

Please comment below if you want to work on this or in case of any question/suggestion.

Mentors for EvalAI-ngx Project (@aka-jain @spyshiv @shivaniprakash95 @RishabhJain2018 @deshraj) will try to respond to this thread ASAP so that you are not blocked.

### PR DESCRIPTION
**Pull request by live-wire**
This is supposed to close #59 
The page is accessible at path `/contact`

This PR is dependent on the form-component PR #70 

> Header will look better once #61 is merged.

Please have a look @RishabhJain2018 @shivaniprakash95 @aka-jain @spyshiv 🍔

### CODE REVIEW

**Comment by live-wire** [comment]
contact
This is how the page will look ☝️ 
> This also contains form validation logic (added to Global Service).
> This PR is dependent on #70 . Please merge that before this. 

Please have a look @aka-jain 🌯 
cc: @RishabhJain2018 @spyshiv

**Comment by live-wire** [comment]
Added the API request to this. Please merge #70 before this.
This also contains common code for the API service which will be required by other pages as well. Please review it soon 🍕 


cc: @aka-jain @spyshiv @RishabhJain2018 @shivaniprakash95 @deshraj

File: src/app/contact/contact.component.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      <br>
+      <div class="contact-form-elements">
+        <app-input [label]="'Name'" [isRequired]="true" #formgroup></app-input>
+        <app-input [label]="'Email'" [isRequired]="true" [type]="'email'" #formgroup></app-input>
+        <app-input [label]="'Message'" [isRequired]="true" #formgroup></app-input>
+        <span class="btn btn-fire" (click)="formValidate('formgroup')">SEND</span>
+        <div class="contact-links">
+          <table>
+            <tr>
+              <td><img src="assets/images/fb.png" /></td>
```
Comment (RishabhJain2018) [inline src/app/contact/contact.component.html]: Please add links to cloudcv fb [page](https://www.facebook.com/mlcloudcv/)

File: src/app/contact/contact.component.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+              <td><img src="assets/images/git.png" /></td>
```
Comment (RishabhJain2018) [inline src/app/contact/contact.component.html]: Here is the github link https://github.com/Cloud-CV

File: src/app/contact/contact.component.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+              <td><img src="assets/images/twitter.png" /></td>
```
Comment (RishabhJain2018) [inline src/app/contact/contact.component.html]: Here is the twitter link https://twitter.com/project_cloudcv

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]
Apart from the links in header, it works perfectly!

**Comment by live-wire** [comment]
Thanks for the links @RishabhJain2018 😄 I made the changes you suggested. 
Please have a look 🔢

**Comment by RishabhJain2018** [comment]
@live-wire Can you please check why the travis build is failing.

**Comment by RishabhJain2018** [comment]
@live-wire I think we should also display a toast notification to the user saying some like `we have received your query and will reply soon`.
Please refer to the current version of EvalAI.

**Comment by live-wire** [comment]
@RishabhJain2018 I added a `TODO` for that in this PR.

Custom Toasts are a part of #77 and the toast you mentioned is added there 😄

**Review by RishabhJain2018 — APPROVED** [review]
LGTM :)
@aka-jain @spyshiv Can we merge this PR ?

**Comment by RishabhJain2018** [comment]
@live-wire More conflicts :D

**Comment by live-wire** [comment]
I'll resolve these right after #66 is merged 💃

**Comment by RishabhJain2018** [comment]
@live-wire Please resolve the merge conflicts here.

**Comment by live-wire** [comment]
On it!

**Comment by RishabhJain2018** [comment]
Closing this since it is already merged.

**Pull request closed without merging**